misk-hibernate/src/test/kotlin/misk/hibernate/HibernateTest.kt

```diff
@@ -0,0 +1,27 @@
+package misk.hibernate
+
+import org.hibernate.mapping.Property
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+internal class HibernateTest {
+  @Test
+  fun inheritedFieldTest() {
+    val parentProperty = Property()
+    parentProperty.name = "parentVal"
+    val childProperty = Property()
+    childProperty.name = "childVal"
+
+    assertEquals("parentVal", field(Parent::class.java, parentProperty).name)
+    assertEquals("parentVal", field(Child::class.java, parentProperty).name)
+    assertEquals("childVal", field(Child::class.java, childProperty).name)
```

Comment:
can we have a failing case as well?

Reply:
There isn't really a failing case per-se. The `assertEquals` is just paranoia to confirm that it returned the right property. If the property doesn't exist, it's an exception.

Reply:
to clarify, that's what I mean by "failing"; a case that throws an exception

Reply:
(and asserting that an exception is indeed thrown)

Reply:
Ok, done

Reply:
thanks!